include/ttmlir/Dialect/TTNN/IR/TTNNOps.td

```diff
@@ -399,6 +399,28 @@ def TTNN_ReshapeOp : TTNN_NamedDPSOp<"reshape"> {
     let hasVerifier = 1;
 }
 
+def TTNN_SliceOp: TTNN_NamedDPSOp<"slice"> {
```

Comment:
I don't think that `ttnn::slice` op is a DPS op, so we shouldn't make it a DPS op in TTNN dialect, though it should stay DPS in TTIR. @mtopalovicTT fixed this for a couple ops recently, you can check his PR for details: #857.

Reply:
Hi sale tnx for heads up, it is dps op we checked it yesterday :D

Reply:
Just saw the comment... I was on an old version where it's not, apologies!

<img width="479" alt="image" src="https://github.com/user-attachments/assets/dc853914-488b-437b-b89f-1893be88c098">